docs/architecture/adr/0023-integration-identifiers.md

```diff
@@ -0,0 +1,89 @@
+---
+adr: "0023"
+status: Proposed
+date: 2024-10-22
+tags: [server]
+---
+
+# 0023 - Identifying Integrated Clients
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
```

Comment:
I don't feel these problems are specific. It's not clear to me how we plan on using these registrations. I think we need to describe what the real benefits of this monitoring and limitations would be.